eldritch/test_unique_items.py

```diff
@@ -0,0 +1,643 @@
+
+  def testSingleSanity(self):
+    self.state.event_stack.append(events.Loss(self.char, {"sanity": 1}))
+    statue = self.resolve_to_usable(0, "Obsidian Statue0")
+    self.state.event_stack.append(statue)
+    self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 5)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testMultipleSanity(self):
+    self.state.event_stack.append(events.Loss(self.char, {"sanity": 2}))
+    statue = self.resolve_to_usable(0, "Obsidian Statue0")
+    self.state.event_stack.append(statue)
+    self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 5)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testSingleStamina(self):
+    self.state.event_stack.append(events.Loss(self.char, {"stamina": 1}))
+    statue = self.resolve_to_usable(0, "Obsidian Statue0")
+    self.state.event_stack.append(statue)
+    self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testMultipleStamina(self):
+    self.state.event_stack.append(events.Loss(self.char, {"stamina": 2}))
+    statue = self.resolve_to_usable(0, "Obsidian Statue0")
+    self.state.event_stack.append(statue)
+    self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testSanityAndStamina(self):
+    self.state.event_stack.append(events.Loss(self.char, {"stamina": 2, "sanity": 2}))
+    statue = self.resolve_to_usable(0, "Obsidian Statue0")
+    self.state.event_stack.append(statue)
+    choice = self.resolve_to_choice(events.MultipleChoice)
+    choice.resolve(self.state, "Stamina")
+    self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+    self.assertEqual(self.char.sanity, 3)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testLossIsValueType(self):
+    die = events.DiceRoll(self.char, 1)
+    loss = events.Loss(self.char, {"stamina": values.Calculation(die, "successes")})
+    self.state.event_stack.append(events.Sequence(
+        [die, loss], self.char
+    ))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      statue = self.resolve_to_usable(0, "Obsidian Statue0")
+      self.state.event_stack.append(statue)
+    self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+    self.assertNotIn(self.statue, self.char.possessions)
+
+  def testLossIsZeroValueType(self):
+    die = events.DiceRoll(self.char, 1)
+    loss = events.Loss(self.char, {"stamina": values.Calculation(die, "successes")})
+    self.state.event_stack.append(events.Sequence(
+        [die, loss], self.char
+    ))
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=3)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.stamina, 5)
+    self.assertIn(self.statue, self.char.possessions)
+
+  def testJewelryAndStatue(self):
```

Comment:
Be sure to test both directions - if the obsidian statue is used, the jewelry should become unusable.